rts/Sim/Misc/Wind.cpp

```diff
@@ -120,7 +121,14 @@ void EnvResourceHandler::Update()
 		curWindDir = curWindVec;
 		curWindVec = curWindDir * (curWindStrength = std::clamp(curWindStrength, minWindStrength, maxWindStrength));
 
-		for (const int unitID: newGeneratorIDs) {
+		if (const auto& wcrp = modInfo.windChangeReportPeriod; wcrp > 0 && gs->frameNum % wcrp == 0) {
```

Comment:
[2/3] ... and moved these two outside the `else` clause into the outer scope ...